collect_app/src/main/java/org/odk/collect/android/tasks/InstanceServerUploader.java

```diff
@@ -103,7 +103,17 @@ protected Outcome doInBackground(Long... instanceIdsToUpload) {
 
             String urlString = getURLToSubmitTo(instance, deviceId, completeDestinationUrl);
 
-            if (!uploadOneSubmission(instance, urlString, uriRemap, outcome)) {
+            SubmissionUploadResult result = uploadOneSubmission(instance, urlString, uriRemap);
```

Comment:
Please note that this structure gets changed in 4d89018 to better match the way GSheets uploading works.